src/js/view/utils/views-manager.js

```diff
@@ -56,11 +77,19 @@ document.addEventListener('webkitvisibilitychange', onVisibilityChange);
 window.addEventListener('resize', scheduleResponsiveRedraw);
 window.addEventListener('orientationchange', scheduleResponsiveRedraw);
 
+if (OS.android && Browser.chrome) {
+    window.screen.orientation.addEventListener('change', onOrientationChange);
+}
+
 window.addEventListener('beforeunload', () => {
     document.removeEventListener('visibilitychange', onVisibilityChange);
     document.removeEventListener('webkitvisibilitychange', onVisibilityChange);
     window.removeEventListener('resize', scheduleResponsiveRedraw);
     window.removeEventListener('orientationchange', scheduleResponsiveRedraw);
+
+    if (OS.android && Browser.chrome) {
```

Comment:
At this point you don't need to check the user-agent. Just make sure `window.screen.orientation` is defined (see above).